doc/src/devdocs/ast.md

```diff
@@ -108,7 +108,7 @@ Doc string syntax:
 f(x) = x
 ```
 
-parses as `(macrocall (|.| Core '@doc) (line) "some docs" (= (call f x) (block x)))`.
+parses as `(macrocall @doc (line) "some docs" (= (call f x) (block (line) x)))`.
```

Comment:
I think this was more correct before. Usually line nodes are implicit, and not represented in the devdocs here.
```suggestion
parses as `(macrocall (core @doc) (line) "some docs" (= (call f x) (block x)))`.
```
But `(core .)` is also internal shorthand for `(|.| Core '@doc)`, so maybe we shouldn't change this one at all
```suggestion
parses as `(macrocall (|.| Core '@doc) (line) "some docs" (= (call f x) (block x)))`.
```